### DISCUSSION TITLE
If I want to create multiple pages under the manage file, I need to create multiple directories and then create multiple pages. tsx separately?

### DISCUSSION THREAD

**Discussion by bbhxwl** [vercel/next.js#69826] [App Router]
If I want to create multiple pages under the manage file, I need to create multiple directories and then create multiple pages. tsx separately?

This is very troublesome. Is there a simple method?

<img width="377" alt="image" src="https://github.com/user-attachments/assets/f5958800-b6de-4954-b1ac-4c5007c0c9f3">

**Comment by Juneezee** [comment]
> I need to create multiple directories and then create multiple pages. tsx separately?

Yes. Note it is `page.tsx`, not `pages.tsx`.

> This is very troublesome. Is there a simple method?

No because Next.js uses a file-system based router where folders are used to define routes.
